python/src/aiconfig/editor/client/src/components/prompt/RunPromptButton.tsx

```diff
@@ -33,7 +33,7 @@ export default memo(function RunPromptButton({
       disabled={disabledOrReadOnly}
       p="xs"
       size="xs"
-      className="runPromptButton"
+      className={`runPromptButton ${readOnly ? "runPromptButtonReadOnly" : ""}`}
```

Comment:
Ok sorry just to be clear, it's specifically this line that causes the prompt button to be vertically centered instead of at the top right? https://github.com/lastmile-ai/aiconfig/blob/c571904edfafd878e60c81962a85b3f9f7de5125/python/src/aiconfig/editor/client/src/themes/GradioTheme.ts#L234

And similar to React where we can overwrite components with spreading ex: 
```
{
  ...stuff_here 
  margin: <overrideMarginValue>
}
```

By declaring `.runPromptButton.runPromptButtonReadOnly` second it will override the previous `marginTop` value defined in `runPromptButton`?

Reply:
Yes, specifically the 33px.

> By declaring .runPromptButton.runPromptButtonReadOnly second it will override the previous marginTop value defined in runPromptButton

Yes and no -- it will override the value, but not due to the ordering but because of the specificity: it applies the styles to any element with both `.runPromptButton` AND `.runPromptButtonReadOnly` classes, whereas the `runPromptButton` selector styles are only applied to an element with that class. Since the button has both classes, the selector will override any conflicting styles